### PR TITLE
DOCS 3.0 : fixing external links including those from api: tags

### PR DESCRIPTION
**Pull request by SpiritLevel**
and a few more links fixed
comment out broken link

I think we're almost done with 3.0 :)

### CODE REVIEW

File: docs/en/changelogs/beta/3.0.0-beta1.md [hunk 1]
```diff
@@ -410,8 +410,8 @@ when using deprecated functionality (through the new `Deprecation` class).
  * 2012-03-08 [9517763](https://github.com/silverstripe/sapphire/commit/9517763) Fixed PHP notice in ModelAdmin (Ingo Schommer)
  * 2012-03-08 [7d956de](https://github.com/silverstripe/sapphire/commit/7d956de) SSF-25 : correct inline document (Normann Lou)
  * 2012-03-08 [72101c2](https://github.com/silverstripe/sapphire/commit/72101c2) SSF-106 - fixed position of pagination on datagrid. (jbridson)
- * 2012-03-08 [2095743](https://github.com/silverstripe/sapphire/commit/2095743) SSF-106 - edited the SCSS so that measurements use increments of $gf_gridy/x (jbridson)
- * 2012-03-08 [349794a](https://github.com/silverstripe/sapphire/commit/349794a) SSF-106 added class="text" to pagination input field for styling (jbridson)
+ * 2012-03-08 `[2095743](https://github.com/silverstripe/sapphire/commit/2095743)` SSF-106 - edited the SCSS so that measurements use increments of $gf_gridy/x (jbridson)
```
Comment (tractorcow) [inline docs/en/changelogs/beta/3.0.0-beta1.md]: Why are we randomly back-tick escaping this link? It works fine.

**Comment by tractorcow** [comment]
All of those links are fine though...

**Comment by SpiritLevel** [comment]
@tractorcow Hmmm...for <= 3.1, I have just been trusting that my link checker is reporting errors properly. For 3.2 and above, I was planning on being far more diligent in checking the links. I'll triple check how my link checker is configured. Maybe it is a simple timeout setting ? I am using a combination of LinkChecker and KLinkStatus.

**Comment by SpiritLevel** [comment]
Yup. Linkchecker has open issues around checking external links and with ssl. I'll stick with KLinkStatus for now :) I was thinking of looking at Selenium...anyone have other suggestions ?

**Comment by tractorcow** [comment]
Do them all manually, but with a ball mouse. ;P

**Comment by SpiritLevel** [comment]
Riiiigggghhttt...like cleaning a floor with a toothbrush ? Is this some kind of SS hazing ritual I have to endure ?

**Comment by camfindlay** [comment]
^ +1 (lolz) 
computer-mouse-999421_960_720

**Comment by SpiritLevel** [comment]
I'll stick with KLinkStatus, if you all don't mind!!!

Anyhow, sorted out the incorrect reporting of broken links. I'll go back through and remove unnecessary backticking :)

**Pull request closed without merging**